fastlane/docs/FAQs.md

```diff
@@ -21,6 +21,24 @@ If you use `rvm`, try the following
 rvm osx-ssl-certs update all
 ```
 
+### Fastlane is slow (to start)
+
+Fastlane uses rubygems and [rubygems is known to be slow itself](http://www.sitepoint.com/rubygems-slow/)
```

Comment:
I didn't mean it as harsh. I found that particular article very enlightening :) Should I change the wording ?